src/components.css

```diff
@@ -97,23 +97,32 @@ details,
 }
 
 .breadcrumbs[aria-label] {
-	font-family: var(--secondary-font);
+
+    font-family: var(--secondary-font);
+
+    &:has([aria-current="page"]) {
+      --separator: var(--breadcrumb-page);
+    }
+    &:has([aria-current="step"]) {
+      --separator: var(--breadcrumb-step);
+    }
 
     & ul, & ol {
-        list-style: none;
         padding-inline-start: 0;
     }
+    & :is(ul, ol):not([type]) {
+        list-style: none;
+    }
 
     & li {
-        display: inline;
-        &+li::before {
-            content: ' / ' / '';
```

Comment:
@dz4k I didn't quite understand the intention of line 110. Is there something I need to be aware of, or is removing it fine?

Reply:
It's to prevent screenreaders from reading the `/` out loud.
https://developer.mozilla.org/en-US/docs/Web/CSS/content#syntax